tests/app/questionnaire/conftest.py

```diff
@@ -569,6 +569,69 @@ def mock_relationship_collector_schema():
     }
 
 
+@pytest.fixture()
+def section_with_custom_summary():
```

Comment:
Unrelated to this but we need to look at a better method of organising fixture partials as keeping large chunks of json in conftest files for single tests is getting unmanageable.

Reply:
Card created on backlog :+1: